product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/getattribute.mdx

```diff
@@ -26,7 +26,7 @@ The namespace URI.
 
 ## Examples
 
-This example creates a new `DOMDocument` named `l_domdoc`, and a `DOMElement` named `elem` with tag name “Departments”. It then adds an attribute to the `DOMElement` named ‘value’. The example turns both `l_domdoc` and `elem` to `DOMNodes` to append `elem` as a child of `l_domdoc`. Finally, it converts `l_domdoc` to `XMLType`.
+This example creates a new `DOMDocument` named `l_domdoc`, and a `DOMElement` named `elem` with tag name “Departments”. It adds an attribute to the `DOMElement` named "value" and appends the `DOMElement` as a child of the `DOMDocument`. Finally, it converts the `DOMDocument` to `XMLType`.
```

Comment:
It adds an attribute to the `DOMElement` named "attr" having a value "value". May be there's no need to mention that the DOMDocument is converted to XMLType. But please feel free to keep it if you think otherwise.